hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java

```diff
@@ -469,95 +463,9 @@ public static ByteBuffer convertKvToByteBuffer(List<KeyValue> keyValues,
     return result;
   }
 
-  /************************ get/set ***********************************/
-  public RedundantKVGenerator setCommonPrefix(byte[] prefix){
-    this.commonPrefix = prefix;
-    return this;
-  }
-
-  public RedundantKVGenerator setRandomizer(Random randomizer) {
-    this.randomizer = randomizer;
-    return this;
-  }
-
-  public RedundantKVGenerator setNumberOfRowPrefixes(int numberOfRowPrefixes) {
-    this.numberOfRowPrefixes = numberOfRowPrefixes;
-    return this;
-  }
-
-  public RedundantKVGenerator setAveragePrefixLength(int averagePrefixLength) {
-    this.averagePrefixLength = averagePrefixLength;
-    return this;
-  }
-
-  public RedundantKVGenerator setPrefixLengthVariance(int prefixLengthVariance) {
-    this.prefixLengthVariance = prefixLengthVariance;
-    return this;
-  }
-
-  public RedundantKVGenerator setAverageSuffixLength(int averageSuffixLength) {
-    this.averageSuffixLength = averageSuffixLength;
-    return this;
-  }
-
-  public RedundantKVGenerator setSuffixLengthVariance(int suffixLengthVariance) {
-    this.suffixLengthVariance = suffixLengthVariance;
-    return this;
-  }
-
-  public RedundantKVGenerator setNumberOfRows(int numberOfRows) {
-    this.numberOfRows = numberOfRows;
-    return this;
-  }
-
-  public RedundantKVGenerator setChanceForSameQualifier(float chanceForSameQualifier) {
-    this.chanceForSameQualifier = chanceForSameQualifier;
-    return this;
-  }
-
-  public RedundantKVGenerator setChanceForSimilarQualifier(float chanceForSimiliarQualifier) {
-    this.chanceForSimilarQualifier = chanceForSimiliarQualifier;
-    return this;
-  }
-
-  public RedundantKVGenerator setAverageQualifierLength(int averageQualifierLength) {
-    this.averageQualifierLength = averageQualifierLength;
-    return this;
-  }
-
-  public RedundantKVGenerator setQualifierLengthVariance(int qualifierLengthVariance) {
-    this.qualifierLengthVariance = qualifierLengthVariance;
-    return this;
-  }
-
-  public RedundantKVGenerator setColumnFamilyLength(int columnFamilyLength) {
-    this.columnFamilyLength = columnFamilyLength;
-    return this;
-  }
-
   public RedundantKVGenerator setFamily(byte[] family) {
     this.family = family;
     this.columnFamilyLength = family.length;
     return this;
   }
-
-  public RedundantKVGenerator setValueLength(int valueLength) {
-    this.valueLength = valueLength;
-    return this;
-  }
-
-  public RedundantKVGenerator setChanceForZeroValue(float chanceForZeroValue) {
-    this.chanceForZeroValue = chanceForZeroValue;
-    return this;
-  }
-
-  public RedundantKVGenerator setBaseTimestampDivide(int baseTimestampDivide) {
-    this.baseTimestampDivide = baseTimestampDivide;
-    return this;
-  }
-
-  public RedundantKVGenerator setTimestampDiffSize(int timestampDiffSize) {
-    this.timestampDiffSize = timestampDiffSize;
-    return this;
-  }
```

Comment:
Not used?

Reply:
Doesn't seem to be the case. Didn't find any usages.

Reply:
Ok. Good.